testing/opae-c/test_init_c.cpp

```diff
@@ -227,7 +227,16 @@ class init_ase_cfg_p : public ::testing::TestWithParam<const char*> {
     strcpy(tmpfile, "opae_ase.cfg.XXXXXX");
     close(mkstemp(tmpfile));
     src_cfg_file_ = cfg_dir + std::string("/") + std::string(tmpfile);
-    rename(OPAE_ASE_CFG_SRC_PATH, src_cfg_file_.c_str());
+    struct stat st;
+    // check if the file exists or not
+    if (!stat(OPAE_ASE_CFG_SRC_PATH, &st)) {
```

Comment:
If either the stat or the rename fails, is it still valid to run the tests?

Reply:
This is the source location. If user installs the OPAE, they can search for other locations for config file.

Reply:
If there is any possibility of overwriting the user's config file, ie if stat passes but rename fails, then we should stop immediately.

Reply:
Make it fail now.